### ISSUE TITLE
can i use patternfly With angular 7?

### ISSUE THREAD

**Issue by AbdelrahmanSayedMohamed** [patternfly/patternfly-ng#457]

**Comment by LarsLuz** [comment]
Yes, you can. It will complain about peerDependencies but it will work.

**Comment by dlabrecq** [comment]
PatternFly-ng is built with Angular 6 and Typescript 2.9.2, but it works with Angular 7. I have not tried it with Angular 8, but believe 8 supports Typescript 3.2, while Angular 7 supports 3.1?

**Comment by mprahl** [comment]
@dlabrecq is it on the roadmap to update patternfly-ng to not complain about peerDependencies?

**Comment by dlabrecq** [comment]
I created a PR to update the peer dependencies.
https://github.com/patternfly/patternfly-ng/pull/462

A couple caveats, tho.

1. PatternFly-ng is not compatible with the dropdown of ngx-bootstrap v3.3.0. Tested v3.0.0-v3.2.0 and they worked fine.

![Screen Shot 2019-06-11 at 3 01 19 PM](https://user-images.githubusercontent.com/17481322/59301197-11cdea00-8c5f-11e9-8bff-84ae6c30f894.png)

2. The following may also need to be added to the options of angular.json

```
"options": {
  "preserveSymlinks": true,
```

See: https://stackoverflow.com/questions/53833985/no-provider-for-viewcontainerref-while-using-ng-packagr-component
